_replication/scheduling/advanced-cron-scheduling.md

```diff
@@ -0,0 +1,334 @@
+
+Using Advanced Scheduling, you can:
+
+1. **Run reports on a schedule**. For example: You need reports for a 9:00AM meeting you have every day. You could create a replication schedule that runs a replication job every night at 12:00AM, which would ensure fresh data is available in the morning when you need it. Reports could be scheduled to run at 7:00AM, giving you time before your meeting to review the data and resolve replication issues, should any arise.
+
+2. **Whitelist hours for starting data extractions.** For example: You know that your production database is under heavy load from 1:00PM to 5:00PM. To avoid adding additional load, you can create a schedule that prevents extractions from starting during this time. **Note**: An extraction may run over into "blackout" hours as the Advanced Scheduler only controls the times jobs **start**. [See the Limitations section for more info](#limitations).
+
+3. **Reduce your row usage**. Only scheduling data extractions when you need them can not only reduce load on data sources and your destination, it can reduce your overall row usage in Stitch.
+
+4. **Reduce the re-replication of data.** Because replication scheduling applies to all selected tables in an integration, tables using Full Table Replication will replicate in full each time a replication job runs. Reducing the number of replication jobs overall will decrease the number of times the same record is replicated.
+
+---
+
+## Advanced Scheduling basics {#basics}
+
+Stitch's Advanced Scheduler feature uses cron scheduling to create replication schedules. In this section, we'll:
+
+- [Introduce you to cron and cron expressions](#intro-to-cron)
+- [Describe cron expression syntax](#con-expression-syntax)
+- [Explain how to format a cron expression](#cron-expression-field-values)
+- [Explain how specifying days works](#specifying-days)
+
+### Introduction to cron {#intro-to-cron}
+
+So, what's cron? **Cron** is a time-based scheduler used in Unix-like operating systems such as Mac OS, Linux, etc. Tasks, or jobs, created through cron are called **cron jobs**.
+
+To create a cron job - in this case, an integration's replication schedule - a **cron expression** is used. A cron expression describes the details of the schedule, and when combined, translates to the schedule Stitch will use to replicate data from the integration.
+
+Using a cron expression, you can create a replication schedules such as _"At 12:00AM every day"_ or _"At 7:00PM every Monday, Wednesday, and Friday"_.
+
+### Cron expression syntax {#cron-expression-syntax}
+
+A cron expression in Stitch is made up of five fields that describe the elements of the schedule, separated by spaces.
+
+Fields in the expression must be in the following order, and an expression must have all five fields to be considered valid:
+
+```conf
+[minute] [hour] [day of month] [month] [day of week]
+```
+
+### Formatting cron expressions {#cron-expression-field-values}
```

Comment:
I find this section to be kind of confusing. :(

For instance the sentence:

```
The fields in a cron expression can either contain an asterisk (<code>*</code>), a list of integers separated by commas (ex: <code>1,2,3</code>), or an inclusive range, denoted by two numbers separated by a hyphen (ex: <code>1-5</code>). Fields may also contain both a range and a list.
```

sounds to me to indicate that each field can either be a single integer, a list of single integers, or a single range of integers. Also a field can contain a list/range pair.

This is different than the posix standard reads where each field can be an `*` or an element or a list of elements where an element is a single integer or an inclusive range of single integers. That properly captures that if you have an `*` it's all you can have, if you have an element it can be a single integer or integer range, and if you have a list it can be a list of any mixture of single integers or integer ranges all together.

I may only feel this way because I've been so deep on the posix specification for so long. :)

Reply:
Could be! I think I'll throw this one in Slack and see how others interpret it.

Reply:
Actually, I reworked this a bit:

```
The fields in a cron expression can contain:

- An asterisk (`*`), or
- A single integer (ex: `5`), or
- A list of integers separated by commas (ex: `1,2,3`), or
- An an inclusive range, denoted by two numbers separated by a hyphen (ex: `1-5`), or
- A combination of lists and inclusive ranges (ex: `1,5,10-20`)

Refer to the [Example replication schedules section](link) for examples, or the **Special characters tab** for more info on lists and ranges.
```

I also added clarification to the Special characters section that explains asterisks:

> **Note**: If used, an asterisk can be the only value in the field. For example: `*,30` would result in an error.

Reply:
Stop me at any point if I'm being too nit-picky:

How 'bout:

```
The fields in a cron expression can contain:

- An asterisk (`*`), or
- A single integer (ex: `5`), or
- An inclusive range, denoted by two numbers separated by a hyphen (ex: `1-5`), or
- A list of any number of single integers and inclusive ranges separated by commas (`,`) (ex: `1,5,10-20,25,55-59`)
```